bl-user-setup

```diff
@@ -18,15 +18,15 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # USER="$1" USER is exported by lightdm
-[[ "$USER" = root || ! -d /home/"$USER" ]] && { echo "$0: variable USER has not been set correctly." >&2; exit 1;}
-
-[ -f "/home/$USER/.config/bunsen/bl-setup" ] && exit 0
+HOME_DIR=$( getent passwd $USER | cut -d: -f6 )
+USR_GRP="$( id -ng $USER )"
```

Comment:
Good point, the primary group might be different for LDAP or AD authenticated users.